electricitymap/contrib/config/__init__.py

```diff
@@ -24,6 +24,37 @@
 ZONES_CONFIG = read_zones_config(CONFIG_DIR)
 EXCHANGES_CONFIG = read_exchanges_config(CONFIG_DIR)
 
+EU_ZONES = [
+    "AT",
+    "BE",
+    "BG",
+    "CY",
+    "CZ",
+    "DE",
+    "DK",
+    "EE",
+    "ES",
```

Comment:
What about the smaller zones, i.e `FR-COR` etc?

Reply:
Do we want to include them here?

Reply:
good question, i think in the future if we want to aggregate data we will refer to continental EU and it might be misleading to add the smaller zones. 
We can make 2 lists though

Reply:
Do the data source we will use to compare it have data for EU28 (before GB exited) if so we might want to account for that as well?

Reply:
https://iea.blob.core.windows.net/assets/c5b1e844-3f51-4d56-b395-87d7db87cadd/Monthlyelectricitystatistics_Documentation_2023.pdf